pkg/cluster-runtime/apps.go

```diff
@@ -78,9 +76,9 @@ func (i *AppInstaller) Install(master0 net.IP, cmds []string) error {
 
 func (i AppInstaller) LaunchClusterImage(master0 net.IP, launchCmds []string) error {
 	var (
-		cmds          []string
-		clusterRootfs = i.infraDriver.GetClusterRootfsPath()
-		ex            = shell.NewLex('\\')
+		cmds    []string
+		appPath = i.infraDriver.GetClusterRootfsPath()
```

Comment:
It looks like we can keep use `clusterRootfs`.

And In my understanding, we may need a new `appRootPath` variable, and it's going to be filepath.Join(clusterRootfs,"applications")

Reply:
yes, we should launch app in their own workspace, thats why i rename this value from "clusterRootfs" to "appRootPath". later we only need to update this func to support it directly.